flow/beta/cli/config/json/config.go

```diff
@@ -0,0 +1,105 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019-2020 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package json
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/onflow/flow-cli/flow/beta/cli/config"
+)
+
+type jsonConfig struct {
+	Contracts jsonContracts `json:"contracts"`
+	Networks  jsonNetworks  `json:"networks"`
+	Accounts  jsonAccounts  `json:"accounts"`
+	Deploys   jsonDeploys   `json:"deploys"`
+}
+
+func (j jsonConfig) transformToConfig() *config.Config {
+	return &config.Config{
+		Contracts: j.Contracts.transformToConfig(),
+		Networks:  j.Networks.transformToConfig(),
+		Accounts:  j.Accounts.transformToConfig(),
+		Deploys:   j.Deploys.transformToConfig(),
+	}
+}
+
+func (j jsonConfig) transformToJSON(config *config.Config) jsonConfig {
+	return jsonConfig{
+		Contracts: j.Contracts.transformToJSON(config.Contracts),
+		Networks:  j.Networks.transformToJSON(config.Networks),
+		Accounts:  j.Accounts.transformToJSON(config.Accounts),
+		Deploys:   j.Deploys.transformToJSON(config.Deploys),
+	}
+}
+
+// Save saves the configuration to the specified path file in JSON format.
+func Save(conf *config.Config, path string) error {
+	jsonConfig := jsonConfig{}.transformToJSON(conf)
```

Comment:
```suggestion
	jsonConf := jsonConfig{}.transformToJSON(conf)
```

Best to avoid overwriting the existing `jsonConfig` definition